### PR TITLE
feat(ProfilePopup): implement copy-to-clipboard button

### LINKED ISSUE #282: Implement copy-to-clipboard component
This is a variation of copy to clipboard where a text value renders a copy-to-clipboard icon next to it. Often used in modals.

### PR DESCRIPTION
**Pull request by PascalPrecht**
Closes #282

### CODE REVIEW

**Comment by PascalPrecht** [comment]
This one needs https://github.com/status-im/dotherside/pull/8 and https://github.com/status-im/nimqml/pull/4 to land first.

**Comment by PascalPrecht** [comment]
Labelling as "do no merge" until DOtherside and nimqml have been updated

File: ui/app/AppLayouts/Chat/components/ProfilePopup.qml [hunk 1]
```diff
@@ -143,6 +143,15 @@ ModalPopup {
             anchors.topMargin: Theme.smallPadding
         }
 
+        CopyToClipBoardIcon {
+          anchors.top: labelEnsUsername.bottom
+          anchors.left: valueEnsName.right
+          anchors.leftMargin: Theme.smallPadding
+          onClick: function (){
+            chatsModel.copyToClipboard("@emily.stateofus.eth")
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Chat/components/ProfilePopup.qml:151]: Can this be replaced by `chatsModel.copyToClipboard(valueEnsName.text)` ?

Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/components/ProfilePopup.qml:151]: Oh yes sure. Just put it as a placeholder for now, but lemme update this

**Pull request merged**